docs/appconfig/schema-definitions-dependencyendpoint.md

```diff
@@ -104,3 +105,19 @@ The TLS port of the dependent service.
 ### tlsPort Type
 
 `integer`
+
+## apiPath
+
+The top level api path that the app should serve from /api/<apiPath>
+
+
+`apiPath`
+
+-   is required
+-   Type: unknown
```

Comment:
shouldn't the type be `String` ?

Reply:
Auto generated - I'm not going to fix this rabbit hole now